buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1704,6 +1704,20 @@ class BeamModulePlugin implements Plugin<Project> {
       return "${configuration.root}/${configuration.name}:${configuration.tag}"
     }
 
+    project.ext.containerImageTags = {
+      String[] tags
+      if (project.rootProject.hasProperty(["docker-tag-list"])) {
+        tags = project.rootProject["docker-tag-list"].split(',')
+      } else {
+        tags = [
+          project.rootProject.hasProperty(["docker-tag"]) ?
+          project.rootProject["docker-tag"] : project.sdk_version,
```

Comment:
I'm not a huge fan of adding this default vs using `tags = [project.rootProject["docker-tag"]]` and having a caller provide `docker-tag-list=${sdk_version},latest`. Is this specific to work you are planning?

Reply:
We need to add some default anyway in case that `docker-tag` property is not given. Since we already use  `project.sdk_version` as a default for `docker` command, I choose the same default for `dockerTag`.

Reply:
oh, I see - I thought I hadn't seen it in the existing files but I only looked at the golang task. Can we do just the sdk_version tag then?

Reply:
Done.